### ISSUE TITLE
Is this dead?

### ISSUE THREAD

**Issue by nolde** [esbuild/community-plugins#186]
Seems like this plugin database is dead.
Is that the case?

**Comment by rodw** [comment]
For what it's worth I've posted a brute-force (manual) merge of the 15 currently open PRs to this list in a fork at https://github.com/rodw/community-plugins#readme

That is not meant to be a long-term solution either, but it does add a web-facing version of the stuff that's been submitted to the list in the past year at least.

**Comment by evanw** [comment]
No, it's not dead. But I could definitely use a ping sometimes :)

Thanks for the ping. I'll try to process these today.

**Comment by rodw** [comment]
Awesome!  🎉  thanks @evanw

Also thank you so very much for the excellent esbuild project in general. The more I use it the more I value and appreciate it.

(Also BTW I for one strongly encourage you to stick to your singular, cohesive vision for esbuild. Feature bloat and muddy scope is a major issue with many modern JS build tools and much of broader build pipeline ecosystem in general IMO. esbuild's design philosophy is a breath of fresh air.)

---

FWIW I took a look at some of those curated awesome-<whatever> lists to see how they manage this, but it seems like many of them eventually run into this kind of problem too. It takes persistent, tedious effort just to keep up with the raw new submissions, let alone maintain the taxonomy or prune out orphaned/archived/outdated libraries or etc.

I wonder if it might be more sustainable to "crowd-source" some of this by establishing, encouraging and (importantly) directly linking to some kind of some kind of tagging or naming convention that provides a way for plugin contributors to self-organize that "directory" a little. E.g., searching npm just now there seem to be:
 * ~150 public modules with the keyword (tag) ["esbuild-plugin"](https://www.npmjs.com/search?q=keywords:esbuild-plugin)
 * ~360 public modules with the combination of both ["esbuild" and "plugin"](https://www.npmjs.com/search?q=keywords:esbuild,plugin) as independent keywords.

Would it make sense to explicitly tell plugin contributors "make sure to include the keyword `esbuild-plugin` in your package.json" and just link to those search results?

Perhaps that should be done "in addition to" rather than "instead of" a genuinely curated list. I.e. using the keyword search as blessed-by-convention way to get an _exhaustive_ list of plugins while allowing the manually maintained list be more selective (hence shorter and less frequently updated).

**Comment by evanw** [comment]
> Would it make sense to explicitly tell plugin contributors something like "make sure to include the keyword `esbuild-plugin` in your package.json" and just link to those search results?

That's a good suggestion, thanks. This list was obviously a lot smaller when it started a while ago and it hasn't been reconsidered since. I'll work this into esbuild's documentation.

> Perhaps that should be done "in addition to" rather than "instead of" a genuinely curated list. I.e. using the keyword search as blessed-by-convention way to get an _exhaustive_ list of plugins while allowing the manually maintained list be more selective (hence shorter and less frequently updated).

Heads up that this is definitely not a curated list. This is a just list of plugins from people who have taken the time to submit a PR to put something on this list. I'm not willing to commit to curating a list of the best and/or most relevant esbuild plugins.

> It takes persistent, tedious effort just to keep up with the raw new submissions, let alone maintain the taxonomy or prune out orphaned/archived/outdated libraries or etc.

I don't think it's necessarily appropriate to try to decide which plugins seem orphaned/archived/outdated. Plugins are typically little bits of glue code that get a very specific job done. Most of them are written once for that need at that time. I think it's ok for them to be orphaned/archived once used as they have served their purpose, and the person with that need has moved on. But they're also still useful as reference for other people in the future who have the same or similar need, and who either want to use the plugin as-is or use it as a starting point.

Closing this issue as resolved since this list is indeed not dead. But I'll work on the npm keyword suggestion as a follow-up to this issue.
